custom-words.txt

```diff
@@ -2531,6 +2531,9 @@ businessmetadata
 Qualys
 operatorconnect
 servermetrics
+Autorization
```

Comment:
Authorization?

Please fix this in your swagger and examples.

Reply:
fixed